randovania/game_description/trick_documentation.py

```diff
@@ -40,7 +46,10 @@ def _find_tricks_usage_documentation(requirement: Requirement) -> Iterator[tuple
             if it.amount > it.resource.require_documentation_above:
                 documentation_state = TrickUsageState.DOCUMENTED
 
-    if documentation_state == TrickUsageState.DOCUMENTED and requirement.comment is None:
+    undocumented = requirement.comment is None or any(
+        keyword.lower() in requirement.comment.lower() for keyword in EXCLUSION_KEYWORDS
```

Comment:
IIRC `.casefold()`  is the recommended approach to comparing case insetitive?
https://docs.python.org/3/library/stdtypes.html#str.casefold